packages/cactus-plugin-ledger-connector-corda/src/main/typescript/web-services/start-flow-endpoint-v1.ts

```diff
@@ -0,0 +1,108 @@
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import {
+  Checks,
+  IAsyncProvider,
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+
+import OAS from "../../json/openapi.json";
+import { PluginLedgerConnectorCorda } from "../plugin-ledger-connector-corda";
+
+export interface IStartFlowEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  apiUrl?: string;
+  connector: PluginLedgerConnectorCorda;
+}
+
+export class StartFlowEndpointV1 implements IWebServiceEndpoint {
+  public static readonly CLASS_NAME = "StartFlowEndpointV1";
+
+  private readonly log: Logger;
+  private readonly apiUrl?: string;
+
+  public get className(): string {
+    return StartFlowEndpointV1.CLASS_NAME;
+  }
+
+  constructor(public readonly options: IStartFlowEndpointV1Options) {
+    const fnTag = `${this.className}#constructor()`;
+
+    Checks.truthy(options, `${fnTag} options`);
+    Checks.truthy(options.connector, `${fnTag} options.connector`);
+    this.log = LoggerProvider.getOrCreate({
+      label: "start-flow-endpoint-v1",
+      level: options.logLevel || "INFO",
+    });
+
+    this.apiUrl = options.apiUrl;
+  }
+
+  getAuthorizationOptionsProvider(): IAsyncProvider<IEndpointAuthzOptions> {
+    return {
+      get: async () => ({
+        isProtected: true,
+        requiredRoles: [],
+      }),
+    };
+  }
+
+  public get oasPath(): (typeof OAS.paths)["/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-corda/startFlow"] {
+    return OAS.paths[
+      "/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-corda/startFlow"
+    ];
+  }
+
+  /**
+   * Returns the endpoint path to be used when installing the endpoint into the
+   * API server of Cactus.
+   */
+  public getPath(): string {
+    return this.oasPath.post["x-hyperledger-cactus"].http.path;
+  }
+
+  public getVerbLowerCase(): string {
+    return this.oasPath.post["x-hyperledger-cactus"].http.verbLowerCase;
+  }
+
+  public getOperationId(): string {
+    return this.oasPath.post.operationId;
+  }
+
+  public getExpressRequestHandler(): IExpressRequestHandler {
+    return this.handleRequest.bind(this);
+  }
+
+  public async registerExpress(
+    expressApp: Express,
+  ): Promise<IWebServiceEndpoint> {
+    await registerWebServiceEndpoint(expressApp, this);
+    return this;
+  }
+  async handleRequest(req: Request, res: Response): Promise<void> {
+    const fnTag = "startFlowV1#handleRequest()";
+    const verbUpper = this.getVerbLowerCase().toUpperCase();
+    this.log.debug(`${verbUpper} ${this.getPath()}`);
+    try {
+      if (this.apiUrl === undefined) throw "apiUrl option is necessary";
+      const body = await this.options.connector.startFlow(req.body);
+      res.status(200);
+      res.json(body);
+    } catch (ex) {
+      this.log.error(`${fnTag} failed to serve request`, ex);
+      res.status(500);
+      res.statusMessage = ex.message;
+      res.json({ error: ex.stack });
+    }
```

Comment:
@adrianbatuto Please refactor this error handler to use the handleRestException function instead

Reply:
@adrianbatuto Sorry, what I meant is to fix it in all the endpoint implementations (6 or 7 more places in the code IIRC)